### ISSUE TITLE
Incorrect test result for ordered lists tech/html/ol

### ISSUE THREAD

**Issue by aferreira-deo** [accessibilitysupported/a11ysupport.io#238]
The output of the test for Android + Talkback is incorrect, it should be set to result failed, as is the case for Voiceover:
https://a11ysupport.io/tests/tech__html__ol#assertion-html-li_element-convey_posinset_ol_context--talkback-and_chr
![image](https://user-images.githubusercontent.com/98042888/177130794-74a88ed8-f444-4de1-8291-d562de77d863.png)

I tested on the latest version of Android and indeed the output is just the number 2 which is incorrect, it should announce "2. dogs" at the very least

**Comment by mfairchild365** [comment]
Thanks for pointing this out! I'm re-testing this behavior on the latest version of talkback and voiceover. In both, swiping right first lands on "2", then lands on "dog". So we hear the number 2, swipe right, and then hear "dog". I think this is fine behavior to pass the expectation in this scenario. I'll work on testing a bit more and update the results.
